opp-distribution-service/contracts/testnet/sepolia-to-boba/contracts/SepoliaToBobaBridge.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IBobaStandardBridge {
+    function depositERC20To(address _token, address _to, uint256 _amount, uint256 _l2Gas, bytes calldata _data) external;
+}
+
+interface IWETH {
+    function deposit() external payable;
+    function approve(address spender, uint256 amount) external returns (bool);
+    function transfer(address to, uint256 value) external returns (bool);
+}
+
+contract SepoliaToBobaBridge {
+    IBobaStandardBridge public bobaStandardBridge;
+    IWETH public WETH;
+
+    address public bobaBridgeAddress;
+    address public wethAddress;
+
+    // Constructor to set the Boba Standard Bridge address
+    constructor() {
+        bobaBridgeAddress = 0x244d7b81EE3949788Da5F1178D911e83bA24E157;
```

Comment:
This requires a little hunting to find these addresses - not always in documentation